libs/wingsdk/src/target-sim/bucket.ts

```diff
@@ -17,8 +17,10 @@ import { BaseResourceSchema } from "../testing/simulator";
 export class Bucket extends cloud.Bucket implements ISimulatorResource {
   private readonly public: boolean;
   private readonly initialObjects: Record<string, string> = {};
+  private scope: Construct
   constructor(scope: Construct, id: string, props: cloud.BucketProps = {}) {
     super(scope, id, props);
+    this.scope = scope
```

Comment:
please remove this unused var :)

Reply:
Hi @tsuf239 

I need **scope** variable from the constructor. So I can use it in the **addFile** method.

![image](https://github.com/winglang/wing/assets/62931444/ade73794-cb45-4944-b4a6-41f84592dff8)

I'm actually **reading the scope value in the addFile method** but I'm still getting this error in codespaces.

Reply:
You are right! you need the scope in libs/wingsdk/src/**cloud**/bucket.ts I asked you to remove the lines from libs/wingsdk/src/**target-sim**/bucket.ts it's not the same file :)

Reply:
OH!
Thank you. Got it.
I have removed it.